index.html

```diff
@@ -0,0 +1,46 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Portfolio</title>
+    <!-- css link -->
+    <link rel="stylesheet" href="./assets/css/main.css" />
+  </head>
+  <body>
+    <!-- navigatin section -->
+    <nav id="navbar">
+      <div class="logo">
+        <a href="#">Eric Mahare</a>
+      </div>
+      <span class="nav_icon">
+        <img src="./assets/images/icons/hamburger.png" alt="" />
+      </span>
+    </nav>
```

Comment:
Good use of `nav` tag here 👍 but as of the [project requirement](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/1_setup.md#project-requirements) this `nav` tag has to be inside the `header` tag along with the `logo` we can use Flexbox for the `header` to display the `logo` and `nav` section in one row. Kindly update that.